easybuild/easyconfigs/v/VTune/VTune-2021.5.0.eb

```diff
@@ -0,0 +1,24 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
```

Comment:
@robert-mijakovic I think it would be helpful to include a comment referring to the need to add `--accept-eula-for=Intel-oneAPI` when building this module.

Reply:
Solved!